runtime/parachains/src/scheduler/tests.rs

```diff
@@ -1017,18 +1018,13 @@ fn availability_predicate_works() {
 		run_to_block(1 + group_rotation_frequency + chain_availability_period, |_| None);
 
 		{
-			let pred = Scheduler::availability_timeout_predicate()
-				.expect("predicate exists recently after rotation");
-
+			let pred = Scheduler::availability_timeout_predicate();
 			let would_be_timed_out = System::block_number() - thread_availability_period;
 
-			assert!(!pred(CoreIndex(0), would_be_timed_out)); // chains can't be timed out now.
-			assert!(pred(CoreIndex(1), would_be_timed_out)); // but threads can.
+			// Chains and threads are handled equally
+			assert!(pred(CoreIndex(0), would_be_timed_out)); // chains can't be timed out now.
+			                                     //assert!(pred(CoreIndex(1), would_be_timed_out)); // but threads can.
```

Comment:
This looks unfinished. We still have `thread_availability_period`, but here you claim chains and threads are handled equally and comment out an assertion.